### PR TITLE
Fix ingesting bug that creates one file per table

### LINKED ISSUE #114: Ingest creates multiple SQLite files, one per compartment
e.g. 
```
backend.sqlite::Nuclei
backend.sqlite::Cells
backend.sqlite::Cytoplasm
```

I wasn't able to reproduce this fully, yet.

### PR DESCRIPTION
**Pull request by shntnu**
This fixes #114 introduced in #108 where the ingest was being done incorrectly (one file was being created per table).

### CODE REVIEW

File: cytominer_database/ingest.py [hunk 1]
```diff
@@ -99,12 +99,12 @@ def into(input, output, name, identifier, skip_table_prefix=False):
             #     deprecated, use inspect.signature() or inspect.getfullargspec()
             warnings.simplefilter("ignore", category=DeprecationWarning)
 
-            target = "{}::{}".format(output, name)
+            target = output
```
Comment (gwaybio) [inline cytominer_database/ingest.py]: is this line required? can we just do `engine = create_engine(output)` in the next line?

**Review by gwaybio — APPROVED** [review]
🎉

did an update to sqlite cause this bug?

File: cytominer_database/ingest.py (hunk 1 above)
Comment (shntnu) [reply cytominer_database/ingest.py]: Sounds good, will do

**Comment by shntnu** [comment]
> did an update to sqlite cause this bug?

Nope, the migration to `pandas` introduced this error, and it had gone unnoticed. I then introduced a simple [test](https://github.com/cytomining/cytominer-database/pull/115/commits/4df6d1d61390b2bf6a2dd1a55f8f918848c401b5) to catch it.

**Pull request merged**